gatsby-node.js

```diff
@@ -271,6 +271,7 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
       createPageFromNode(i18nNode || node, {
         prefix: i18nNode ? '' : locale,
         createPage,
+        disableSEO: !i18nNode,
```

Comment:
originally I had named this because I thought we might wanna disable all SEO for these pages, but after thinking about it I realized that might not be the best option (Google has a way better way for organizing these pages based on locale, so I would want to interrupt that). Maybe, `disableSwiftype` or `disableSwiftypeIndex`?

Reply:
changed it :)